components/dashboard/src/index.tsx

```diff
@@ -57,31 +57,33 @@ const bootApp = () => {
     // Render the App
     ReactDOM.render(
         <React.StrictMode>
-            <GitpodQueryClientProvider>
-                <ConfettiContextProvider>
-                    <UserContextProvider>
-                        <AdminContextProvider>
-                            <PaymentContextProvider>
-                                <LicenseContextProvider>
-                                    <TeamsContextProvider>
-                                        <ProjectContextProvider>
-                                            <ThemeContextProvider>
-                                                <StartWorkspaceModalContextProvider>
-                                                    <BrowserRouter>
-                                                        <FeatureFlagContextProvider>
-                                                            <App />
-                                                        </FeatureFlagContextProvider>
-                                                    </BrowserRouter>
-                                                </StartWorkspaceModalContextProvider>
-                                            </ThemeContextProvider>
-                                        </ProjectContextProvider>
-                                    </TeamsContextProvider>
-                                </LicenseContextProvider>
-                            </PaymentContextProvider>
-                        </AdminContextProvider>
-                    </UserContextProvider>
-                </ConfettiContextProvider>
-            </GitpodQueryClientProvider>
+            <GitpodErrorBoundary>
```

Comment:
Can we re-use the same error-boundary component and render it inside e.g. the "User settings" menu in case that errors? :thinking:

Reply:
Totally, I already tested it, and it works, and leaves the surrounding UI functional, so a user can navigate somewhere else and not be stuck. It's slightly more work than I wanted to include here, so I'll do it as a followup. Am hoping to implement it at the `<Route/>` component level so we can have all of our routes guarded by an error boundary.